### ISSUE TITLE
[🐞] [Website] 500 - Cannot communicate with REPL service worker

### ISSUE THREAD

**Issue by AxDSan** [QwikDev/qwik#6782]
### Which component is affected?

Qwik Playground

### Describe the bug

I am trying to see the runnable examples in the website but most of them if not all of them are returning this in the consoles, and nothing loads... 

> 500 - Cannot communicate with REPL service worker - https://qwik.dev/repl/1m5orwzsm53/

![image](https://github.com/user-attachments/assets/d641ff8c-8526-45e1-aaaf-695ee796a66f)

It would be expectable to have the runnable examples working and readily available instead.

### Reproduction

https://qwik.dev/examples/introduction/hello-world/#f=G000o4mG5EQDAA

### Steps to reproduce

Just visit the link and the examples should act up

### System Info

```shell
System:
    OS: Windows 11 10.0.22635
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 3.56 GB / 15.91 GB
  Binaries:
    Node: 22.1.0 - ~\AppData\Local\fnm_multishells\28520_1723334187722\node.EXE
    Yarn: 1.22.19 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 9.6.2 - ~\node_modules\.bin\npm.CMD
    pnpm: 9.6.0 - ~\node_modules\.bin\pnpm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.8)
    Internet Explorer: 11.0.22621.3566
```

### Additional Information

N/A

**Comment by wmertens** [comment]
Unfortunately this is caused by a range of things but it should clear up after typing something in the code and/or reloading the page.

Would be great if someone could automate the workaround.

**Comment by drk-mtr** [comment]
I wasn't able to get this to clear up after typing something or reloading. I was on this page:

https://qwik.dev/tutorial/qrl/optimizer/

**Comment by wmertens** [comment]
Right sorry a bug snuck in with version detection. Fixed in #6790
